core/src/rpc.rs

```diff
@@ -382,6 +382,77 @@ impl JsonRpcRequestProcessor {
         Ok(result)
     }
 
+    pub fn get_inflation_reward(
+        &self,
+        pubkey: &Pubkey,
+        epoch: Option<Epoch>,
+    ) -> Result<Option<RpcInflationReward>> {
+        let epoch_schedule = self.get_epoch_schedule();
+        let first_available_block = self.get_first_available_block();
+        let epoch = epoch.unwrap_or_else(|| {
+            epoch_schedule
+                .get_epoch(self.get_slot(Some(CommitmentConfig::finalized())))
```

Comment:
A nit, but you could set `let commitment = Some(CommitmentConfig::finalized())` at the start of this method, since you pass it twice. Also would make commitment a little more obvious, if we ever have questions about results around epoch boundaries.